pkg/sql2pgroll/alter_table.go

```diff
@@ -293,6 +295,37 @@ func convertAlterTableSetColumnDefault(stmt *pgq.AlterTableStmt, cmd *pgq.AlterT
 	return nil, nil
 }
 
+// convertAlterTableDropConstraint convert DROP CONSTRAINT SQL into an OpDropMultiColumnConstraint.
+// Because we are unable to infer the columns involved, placeholder migrations are used.
+//
+// SQL statements like the following are supported:
+//
+// `ALTER TABLE foo DROP CONSTRAINT constraint_foo`
+// `ALTER TABLE foo DROP CONSTRAINT IF EXISTS constraint_foo`
+// `ALTER TABLE foo DROP CONSTRAINT IF EXISTS constraint_foo RESTRICT`
+//
+// CASCADE is currently not supported and will fall back to raw SQL
+func convertAlterTableDropConstraint(stmt *pgq.AlterTableStmt, cmd *pgq.AlterTableCmd) (migrations.Operation, error) {
+	if !canConvertDropConstraint(cmd) {
+		return nil, nil
+	}
+
+	return &migrations.OpDropMultiColumnConstraint{
+		Up: migrations.MultiColumnUpSQL{
+			"placeholder": PlaceHolderSQL,
+		},
+		Down: migrations.MultiColumnDownSQL{
+			"placeholder": PlaceHolderSQL,
+		},
+		Table: stmt.GetRelation().GetRelname(),
+		Name:  cmd.GetName(),
```

Comment:
I think we miss schema-qualified names here.

Reply:
Ah, good catch